### ISSUE TITLE
Datepicker voiceover safari

### ISSUE THREAD

**Issue by woofton** [OfficeDev/office-ui-fabric-react#5726]
### Bug Report 
 
- __Package version(s)__: 6.26.0
- __Browser and OS versions__: safari
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No) No
 
Requested priority: (Blocking, High, Normal, Low) High
 
Products/sites affected: (if applicable)
 
#### Describe the issue:
 
When using voiceover within Safari, the individual dates do not announce anything.  Those users who are using Safari and using the voiceover will not be able to know which date they are selecting.  This is similar to https://github.com/OfficeDev/office-ui-fabric-react/issues/5339 but this bug is specific to Safari.
 
#### Actual behavior:
 
Using voiceover commands, not able to navigate to the individual dates.  Can use the arrows to move around the individual dates but nothing is announced.
 
#### Expected behavior:
 
Each individual date should announce the date they are, for examples, if I navigate to July 31st, 2018 within the datepicker using the voiceover commands, voiceover should announce "July 31st, 2018"

**Comment by cliffkoh** [comment]
Hi @woofton, #5339 is not exactly the same because it is saying that the DatePicker is not reading the Day although it is reading the Date. 

To make this actionable, please identify the specific manner that the DatePicker is not adhering to W3C accessibility specs as the behavior you're observing on Safari does not repro on Narrator with Edge. Our first and foremost priority is to take a standards-based approach and we will make fixes in cases where we are not adhering to standards.

**Comment by woofton** [comment]
As started in the actual behavior, using voiceover in safari, we are not able to navigate to the individual days and therefore a user is not able to hear "July 30th, 2018" and then "July 31st, 2018".  As a user I will have to blindly select a day, have the calendar close, and then hear the day that was selected and continue this behavior till I get to the day I want.  We are able to navigate to the headers (Friday, Saturday, Sunday, etc.) but no further in the table.  Would you like me to post a video of the behavior?

**Comment by cliffkoh** [comment]
Accessibility support in Fabric aims for correct, standards compliant markup. What user agents and accessibility tools do is up to them.

Please take a look at https://github.com/OfficeDev/office-ui-fabric-react/wiki/Accessibility-Troubleshooting. To make this actionable, we require that you identify the exact way the markup of Calendar is not adhering to W3C accessibility standards.
